tests/init_ico.rs

```diff
@@ -60,6 +61,7 @@ pub fn init(sys: &System) {
 
     init_fungible_token(sys);
     init_ico(sys);
+    sys.mint_to(USER_ID, 100000);
```

Comment:
To be more readable:
```suggestion
    sys.mint_to(USER_ID, 100_000);
```